.github/workflows/deploy.yml

```
@@ -0,0 +1,24 @@
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Deploy to VPS
+        uses: appleboy/ssh-action@v1
+        with:
+          host: ${{ secrets.VPS_HOST }}
+          username: ${{ secrets.VPS_USER }}
+          key: ${{ secrets.VPS_SSH_KEY }}
+          script: |
+            cd ~
+            ./deploy.sh
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)